packages/features/webhooks/lib/cron.ts

```diff
@@ -19,15 +20,32 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
         lte: dayjs().toISOString(),
       },
     },
+    include: {
+      webhook: true,
+    },
   });
 
   // run jobs
   for (const job of jobsToRun) {
     try {
-      await fetch(job.subscriberUrl, {
-        method: "POST",
-        body: job.payload,
-      });
+      if (job.webhook) {
+        const payload = JSON.parse(job.payload);
+        await sendPayload(
```

Comment:
I opt for using sendPayload because it takes care of all the logic involving the secretKey, which is used to send various other webhooks.
believe we should also utilize it for scheduledWebhooks, which we haven't been doing until now.